LICENSE

```diff
@@ -199,4 +199,64 @@
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
-   limitations under the License.
\ No newline at end of file
+   limitations under the License.
+   
+   
+=======================================================================
+Apache Pirk (incubating) Subcomponents:
+
+The Apache Pirk project contains subcomponents with separate copyright
+notices and license terms. Your use of the source code for the these
+subcomponents is subject to the terms and conditions of the following
+licenses.
+
```

Comment:
Just so there is no misunderstanding here, these LICENSE and NOTICE file changes are only to be included in the exe JAR, that bundles all the 3rd-party dependencies together as a convenience for the end user.

In our single project source JAR (```apache-pirk-x.x.x-incubating-sources.jar```), and Pirk-only binary (```apache-pirk-x.x.x-incubating.jar```), these LICENSE/NOTICE modifications must not appear.  There will be different LICENSE and NOTICE files for the source and binary artefacts.

Ref: http://www.apache.org/dev/licensing-howto.html#bundled-vs-non-bundled